src/silx/gui/colors.py

```diff
@@ -238,21 +249,46 @@ def _getColormap(name):
     return _colormap.get_colormap_lut(name)
 
 
+class _Colormappable:
```

Comment:
Base class for objects that interacts with `Colormap` i.e, `ColormapMixIn` plot items: This avoids lazy imports and ease typing.